java/ql/src/experimental/Security/CWE/CWE-117/LogInjection.qll

```diff
@@ -0,0 +1,84 @@
+/**
+ * Provides a taint-tracking configuration for reasoning about untrusted user input used in log entries.
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+
+module LogInjection {
+  string logLevel() {
```

Comment:
Would probably be good to include:
- `debug` (used by e.g. [Log4j1](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Category.html#debug(java.lang.Object)) and [Log4j2](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#debug-java.lang.CharSequence-))
- `log` (used by e.g. [Log4j1](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Category.html#log(org.apache.log4j.Priority,%20java.lang.Object)) and [Log4j2](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#log-org.apache.logging.log4j.Level-java.lang.CharSequence-))